tier/helm/tier/templates/deployment.yaml

```diff
@@ -33,17 +33,26 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - "tier"
+            {{- if .Values.tier.live }}
+            - "-live"
+            {{- end }}
+            - "serve"
+          envFrom:
+            - secretRef:
+                name: {{ include "tier.fullname" . }}-env
           ports:
             - name: http
-              containerPort: 80
+              containerPort: 8080
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: /
+              path: /v1/whoami
```

Comment:
is this a suitable healthcheck?  Not sure if it ends up being an expensive op or not

Reply:
This endpoint does talk to the Stripe API, but it is a true liveness check. If it doesn't work its means its improperly configured.